draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1820,44 +1820,41 @@ IDNA2003 {{RFC3490}}.
+
+While in some instances these purposes may not be privacy-invasive, the potential abuses of tracking imply a need for significant limitations on cookies. However, any such limitations need to be imposed with care, because cookies are widely used for a variety of purposes that are critical to the functioning of a Web site (and the Web).
+
+As a result, this document recommends limited mitigations against the privacy risks associated with cookies below. It is expected that future updates will impose stricter and more well-defined limitations on cookies, based upon experiments now being undertaken in user agents. Note that some functions currently relying on cookies might eventually be supported by separate, targeted mechanisms.
+
+Cookies are not the only mechanism that can be used to track users across sites, so while these mitigations are necessary to improve Web privacy, they are not sufficient on their own.
+
 
 ## Third-Party Cookies {#third-party-cookies}
 
-Particularly worrisome are so-called "third-party" cookies. In rendering an HTML
-document, a user agent often requests resources from other servers (such as
-advertising networks). These third-party servers can use cookies to track the
-user even if the user never visits the server directly. For example, if a user
-visits a site that contains content from a third party and then later visits
-another site that contains content from the same third party, the third party
-can track the user between the two sites.
-
-Given this risk to user privacy, some user agents restrict how third-party
-cookies behave, and those restrictions vary widly. For instance, user agents
-might block third-party cookies entirely by refusing to send Cookie header fields or
-process Set-Cookie header fields during third-party requests. They might take a less
-draconian approach by partitioning cookies based on the first-party context,
-sending one set of cookies to a given third party in one first-party context,
-and another to the same third party in another. Or they might even allow some
-third-party cookies but block others depending on user-agent cookie policy or
-user controls.
-
-This document grants user agents wide latitude to experiment with third-party
-cookie policies that balance the privacy and compatibility needs of their users.
-However, this document does not endorse any particular third-party cookie
-policy.
-
-Third-party cookie blocking policies are often ineffective at achieving their
-privacy goals if servers attempt to work around their restrictions to track
-users. In particular, two collaborating servers can often track users without
-using cookies at all by injecting identifying information into dynamic URLs.
-
-## Cookie policy {#cookie-policy}
+A cookie that is sent to a site other than the one it originated from is known as a "third-party"
+cookie. When rendering a document, a user agent can request resources from sites other than that
```

Comment:
This first sentence is not right, though the remainder of the definition is accurate.  Cookies are always sent back to the entity that set them, but a third-party cookie is a cookie set by a server that is other than the primary entity with which a user interacts.

Maybe: 

> In a Web browsing context, a "third-party" or cross-site cookie is a cookie that is associated with content that is served from a different server than the one that hosts the primary resource (that is, the HTML).  HTML documents can embed content (such as scripts, images, stylesheets, frames) that is served from other domains.  These "third-party" resources [...]

By the way, [Tess recommends "cross-site"](https://tess.oconnor.cx/2020/10/parties).

Reply:
Thanks, I knew terminology was going to be an issue...